### PR TITLE
redbull welcomes its newest member

### PR DESCRIPTION
**Pull request by chirayudesai**
* barbet, Pixel 5a
* Only difference is missing product/lib64/libgdx.so

### CODE REVIEW

**Comment by chirayudesai** [comment]
Extracts RD2A.210605.007 - untested ofc.

**Review by dan-v — APPROVED** [review]
👍

File: barbet/config.json [hunk 1]
```diff
@@ -0,0 +1,772 @@
+{
+  "device-name": "pixel4a5g",
```
Comment (trondd555) [inline barbet/config.json]: Should this be "pixel5a"?

Comment (dan-v) [reply barbet/config.json]: good catch

**Comment by trondd555** [comment]
Builds, boots, and runs on my Pixel 5a.

**Pull request merged**